charts/storage-apps/Chart.yaml

```diff
@@ -18,15 +18,7 @@ annotations:
   artifacthub.io/containsSecurityUpdates: "true"
   artifacthub.io/changes: |
     - kind: changed
-      description: "update csi-driver-smb to 1.14.0"
+      description: "update minio to 5.14.0"
       links:
         - name: Github release
-          url: https://github.com/kubernetes-csi/csi-driver-smb/releases/tag/v1.14.0
-        - name: Github release
-          url: https://github.com/kubernetes-csi/csi-driver-smb/releases/tag/v1.13.0
-        - name: Github release
-          url: https://github.com/kubernetes-csi/csi-driver-smb/releases/tag/v1.12.0
-        - name: Github release
-          url: https://github.com/kubernetes-csi/csi-driver-smb/releases/tag/v1.11.0
-        - name: Github release
-          url: https://github.com/kubernetes-csi/csi-driver-smb/releases/tag/v1.10.0
+          url: https://github.com/minio/minio/releases/tag/RELEASE.2024-12-18T13-15-44Z
```

Comment:
can you also add a link to the release that contains the actual security fix: https://github.com/minio/minio/releases/tag/RELEASE.2024-12-13T22-19-12Z